packages/fast-components-react-msft/src/label/__snapshots__/label.spec.ts.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`label snapshots Label: 0 1`] = `
 <label
-  className="label-0-1-1"
+  className={null}
```

Comment:
@janechu & @nicholasrice should @khamudom remove snapshots?

Reply:
I think there is another task to remove snapshots and It feels that would be the time to remove Label snapshots as well.